packages/react-stream/index.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+// This entry point is intentionally not typed. It exists only for third-party
+// renderers. The renderers we ship (such as React DOM) instead import a named
+// "inline" entry point (for example, `react-stream/inline.dom`). It uses
+// the same code, but the Flow configuration redirects the host config to its
+// real implementation so we can check it against exact intended host types.
+//
+// Only one renderer (the one you passed to `yarn flow <renderer>`) is fully
+// type-checked at any given time. The Flow config maps the
+// `react-stream/inline.<renderer>` import (which is *not* Flow typed) to
+// `react-stream/inline-typed` (which *is*) for the current renderer.
+// On CI, we run Flow checks for each renderer separately.
+
+'use strict';
+
+const ReactFizzRenderer = require('./src/ReactFizzRenderer');
+
+// TODO: decide on the top-level export form.
+// This is hacky but makes it work with both Rollup and Jest.
```

Comment:
Are we still doing this? I think we switched to

```js
export * from './src/ReactFizzRenderer';
```

@gaearon

Reply:
Oh, the other entries do that. Maybe I'm confused.

Reply:
https://github.com/facebook/react/blob/master/packages/react-dom/index.js#L15-L16
https://github.com/facebook/react/blob/master/packages/react-reconciler/index.js#L25-L26

Reply:
I think we do that for packages that are ready to be ESM. But here the actual output will be a function anyway so it doesn’t matter. I guess could work either way.

Reply:
My understanding based on absolutely nothing was that old packages do this because to switch to ESM would be a breaking change. But new packages don't have that restriction.

Reply:
I'd love to fix this though. This is probably really bad for inlining/DCE.